code/modules/hydroponics/goon/grown_food.dm

```diff
@@ -0,0 +1,184 @@
+/obj/item/weapon/reagent_containers/food/snacks/grown/tomatoexplosive
+	name = "seething tomato"
+	desc = "I say to-mah-to, you say KABOOM."
+	icon_state = "explosivetomato"
+	filling_color = "#FF0000"
+	potency = 10
+	plantname = "explosivetomato"
+
+/obj/item/weapon/reagent_containers/food/snacks/grown/tomatoexplosive/throw_impact(atom/hit_atom)
```

Comment:
would still probably be best if these functioned like primed grenades with random fuses--especially considering how easy these will be to produce.